src/BasicInputs/BasicNumberInput.js

```diff
@@ -4,6 +4,7 @@
 import { Grid, TextField, Typography, withStyles } from '@material-ui/core';
 import PropTypes from 'prop-types';
 import React from 'react';
+import NumberFormat from 'react-number-format';
```

Comment:
_react-number-format_ must be added to the dependencies of the @cosmotech/ui module